### ISSUE TITLE
Are there plans to train a model with surface reflectance imagery?

### ISSUE THREAD

**Issue by roserustowicz** [dlr-eoc/ukis-csmask#17]
**Is your feature request related to a problem? Please describe.**
The Sentinel-2 L2A and Landsat Collection 2 products are processed to surface reflectance, while the current version of the UKIS cloud and shadow masking model was trained on top of atmosphere reflectance. Although the model can still be applied to surface reflectance imagery, this is not what it was trained on. It begs the question if a model trained on surface reflectance would perform better than the current TOA model. 

**Describe the solution you'd like**
A UKIS CSMask trained on surface reflectance from the Sentinel 2 L2A and Landsat Collection 2 (4/5/7/8/9) archive.

**Comment by nmandery** [comment]
@MWieland ?

**Comment by MWieland** [comment]
I agree that a comparison between L1C and L2A models would be interesting, however we currently don't have plans (or rather the resources) to support L2A images in ukis-csmask. We simply did not discover the need for this so far, as we work mainly with L1C images. Did you already try to use ukis-csmask on surface reflectance imagery and if so can you provide a rough idea of the performance?

**Comment by MWieland** [comment]
Closing this issue for now due to inactivity.

**Comment by MWieland** [comment]
v1.0.0 is released now and supports both L1C and L2A product levels.
